apiv2/serializers.py

```diff
@@ -46,23 +46,12 @@
 DEFAULT_FIELDS_IN_PACK_DETAIL = None  # Separated by commas (None = all)
 
 
-def get_sound_analyzers_output_helper(sound, fallback_to_db=True):
+def get_sound_analyzers_output_helper(sound):
     analyzers_output = {}
+    sound_analysis_dict = {an.analyzer: an.analysis_data for an in sound.analyses.all()}
```

Comment:
at this point in time, `sound.analyses.all()` has been preloaded. We don't want to do `sound.analyses.filter(xxxx)` because that will trigger a new DB request. Instead turn it into a dict based on key and then get these key items based on the settings value.